polynomial/commitment.go

```diff
@@ -32,6 +32,7 @@ type Polynomial interface {
 type Digest interface {
 	io.WriterTo
```

Comment:
are these two interfaces used? (io.WriterTo, io.ReaderFrom) on Digest?

Reply:
Currently there are not used, but they will be eventually (the commitments need to be passed to the rollup operator/blockchain at some point).